### PR TITLE
Restrict payment updates

### PR DESCRIPTION
**Pull request by turbo124**
PUT route should only update meta data

### CODE REVIEW

File: src/pages/payments/edit/Edit.tsx [hunk 1]
```diff
@@ -72,7 +72,7 @@ export function Edit() {
       }}
     >
       {payment?.client && <ClientCard client={payment.client} />}
-      {payment && <PaymentOverview payment={payment} />}
+      {payment && <PaymentOverview payment={data?.data.data} />}
```
Comment (beganovich) [inline src/pages/payments/edit/Edit.tsx:75]: If we are pushing empty list of invoices/credits into `payment` object, why don't we use it here? What's purpose of the `payment` then?

**Review by beganovich — COMMENTED** [review]
Looks good, just to confirm the changes are the ones we wanted.

File: src/pages/payments/edit/Edit.tsx (hunk 1 above)
Comment (turbo124) [reply src/pages/payments/edit/Edit.tsx:75]: Payment cannot have invoices or credit array in request.

However we need invoices array to display existing invoices in the payment

Comment (beganovich) [reply src/pages/payments/edit/Edit.tsx:75]: That's fine, yes. But we are making `payment` variable essentially `data.data.data` just with empty invoices and then completely ignoring that variable? Why use it at all? 

Why was this change from `payment` to `data.data.data`?

**Comment by beganovich** [comment]
Looks good, thanks :+1:

**Pull request merged**